test/scripts/common.inc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Include file with common parts shared for local and ephemeral
+#
+
+function error {
+    local err=$?
+    printf "ERROR: %s\n" "$*" >&2
+    exit $err
+}
+
+ORG_ID="${ORG_ID:-12345}"
+
+IDENTITY_USER="$( ./tools/bin/xrhidgen -org-id "${ORG_ID}" user -is-active=true -is-org-admin=true -user-id test -username test | base64 -w0 )"
```

Comment:
I don't like the fact that you call the tool unconditionally. You can do

```
user_identity() {
    ${XRHIDGEN} -org-id "${ORG_ID}" user -is-active=true -is-org-admin=true -user-id test -username test | base64 -w0
}
```

and then assign the output on demand with

```
X_RH_IDENTITY=$(user_identity)
```

This works because the command outputs to stdout and `$()` returns stdout for the assignment operator.

Reply:
JFTR: I had to export the function to make it available in the subshell